dom.bs

```diff
@@ -9771,12 +9772,14 @@ method, when invoked, must run these steps:
  <a>throw</a> an "{{InvalidCharacterError!!exception}}" {{DOMException}}.
 
  <li><p>If <a>context object</a>'s <a>token set</a> does not <a for=set>contain</a>
- <var>token</var>, then return.</p>
+ <var>token</var>, return false.</p>
```

Comment:
Please keep "then".

Reply:
Okay, I opted to remove "then" after finding that the phrasing ", return [bool]" was more common than ", then return [bool]" in algorithms throughout the document. Would you like me to change it elsewhere too, in a separate pull request?

Reply:
Yeah, if you want.